README.textile

```diff
@@ -27,3 +27,39 @@ Get @digitalisiertedrucke@, set up the Play application, and run the tests:
 * @cd git/digitalisiertedrucke ; ~/activator-1.3.10-minimal/bin/activator test@
 
 See the @.travis.yml@ file for details on the CI config used by Travis.
+
+h1. Data import
+
+To transform the bzipped Marc-XML source data to JSON and index into an embedded Elasticsearch index, run:
+
+@~/activator-1.3.10-minimal/bin/activator "runMain ImportData"@
```

Comment:
This is not like in lobid-organisations, where we transform and index when the app starts, and the process keeps running after indexing is done. Here, we just run that main class, so it's done when we process finishes and we get back the command line prompt. It also logs to the console for me (both our own app info and some Elasticsearch logging).